Makefile

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 The Service Manager Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: build test
+
+BINDIR ?= bin
+TEST_PROFILE ?= $(CURDIR)/profile.cov
+COVERAGE ?= $(CURDIR)/coverage.html
+PROJECT_PKG = github.com/Peripli/service-broker-proxy-cf
+
+PLATFORM ?= linux
+ARCH     ?= amd64
+
+BUILD_LDFLAGS =
```

Comment:
probably we can by default pass here a script that calculates the current commit or some release tag?